docs/sources/sharing/share-panel.md

```diff
@@ -10,7 +10,13 @@ draft = "true"
 
 # Share a panel
 
-Click a panel title to open the panel menu, then click share in the panel menu to open the Share Panel dialog. Here you have access to a link that will take you to exactly this panel with the current time range and selected template variables. Below are ways to share a panel.
+Click a panel title to open the panel menu, then click share in the panel menu to open the Share Panel dialog. Here you have access to a link that will take you to exactly this panel with the current time range and selected template variables. Here are some of the ways you can share a panel.
+
+## Publish snapshots
+
+You can publish snapshots to your local instance or to [snapshot.raintank.io](http://snapshot.raintank.io). The latter is a free service provided by [Raintank](http://raintank.io), that allows you to publish dashboard snapshots to an external Grafana instance. The same rules still apply, anyone with the link can view it. You can set an expiration time if you want the snapshot to be removed after a certain time period.
```

Comment:
I suggest rewriting this paragraph with the two options as a bulleted list, or break it up some other way so that the paragraph is less than four sentences long and uses simpler language.